htdocs/comm/propal/card.php

```diff
@@ -121,7 +121,9 @@
 $usercancreateorder = $user->rights->commande->creer;
 $usercancreateinvoice = $user->rights->facture->creer;
 $usercancreatecontract = $user->rights->contrat->creer;
-$usercancreateintervention = $user->rights->ficheinter->creer;
+if (!empty($conf->ficheinter->enabled)) {
```

Comment:
Using such condition at each permission test will lead to a lot of useless code. Also here the usercancreateintervention is not more initialized in some cases.
So i added a method user->hasRight(). You can now replace:

$usercancreateintervention = $user->rights->ficheinter->creer;
with 
$usercancreateintervention = $user->hasRight('ficheinter', 'creer');